src/domain/orders/details/address-modal.tsx

```diff
@@ -24,19 +25,17 @@ const AddressModal: React.FC<AddressModalProps> = ({
     defaultValues: { ...address },
   })
 
-  register("country_code")
```

Comment:
what did this do?

Reply:
i don't think it did anything. if i'm not mistaken it was throwing an error, and it's counter to how the register function should be used per the docs https://react-hook-form.com/api/useform/register

Reply:
lolol ok cool, just checking.